src/clientrpc.cpp

```diff
@@ -94,11 +94,67 @@ CClientRpc::CClientRpc ( CClient* pClient, CRpcServer* pRpcServer, QObject* pare
                                             } );
     } );
 
+    /// @rpc_notification jamulusclient/serverListReceived
+    /// @brief Emitted when the server list is received.
+    /// @param {array} params.servers - The server list.
+    /// @param {string} params.servers[*].url - IP Address
+    /// @param {string} params.servers[*].name - Server name
+    /// @param {number} params.servers[*].countryId - Server country code
+    /// @param {string} params.servers[*].city - Server city
+    connect ( pClient->getConnLessProtocol(), &CProtocol::CLServerListReceived, [=] ( CHostAddress /* unused */, CVector<CServerInfo> vecServerInfo ) {
+        QJsonArray arrServerInfo;
+        for ( const auto& serverInfo : vecServerInfo )
+        {
+            QJsonObject objServerInfo{
+                { "url", serverInfo.HostAddr.toString() },
+                { "name", serverInfo.strName },
+                { "countryI", serverInfo.eCountry },
```

Comment:
Is that consistent how we handle countries elsewhere? Also why is it countryI? Not countryId?

Reply:
countryId is how country codes are handled elsewhere, e.g. in clientList.
"countryI" is a type - I will fix.

Reply:
On further consideration (and attempt to use in the field) I think this should be a string holding the country name as text. QLocale is QT specific and JSON-RPC clients may not be QT applications hence this id would be meaningless. I will change it in the PR. I think the country code used in client list should also be changed but that is a breaking change that is not directly related to this PR.